lib/containers/common.pm

```diff
@@ -140,8 +140,11 @@ sub clean_container_host {
         assert_script_run("$runtime rmi --all --force");
     }
     else {
-        assert_script_run("$runtime stop \$($runtime ps -q)", 180) if script_output("$runtime ps -q | wc -l") != '0';
+        my $tmp = "/tmp/podman-ps";
+        assert_script_run "$runtime ps -q | tee $tmp";
```

Comment:
Could be replaced by `$runtime ps -q | xargs -r $runtime stop`?

Reply:
Done. thanks for that